Kill aten_custom_call.

gchanan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#25613 Kill aten_custom_call.**

Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

gchanan committed:
Kill aten_custom_call.

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 1a986196:


* **8/8** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=1a986196a6518d424da51316bad41c4ed2e0c3fd).

### :detective: 8 new failures recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4074929) binary_linux_manywheel_2_7mu_cpu_devtoolset7_build (1/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=78717280) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=78717280))
<details>
<summary>
<code>Dec 27 15:07:19 CMake Error at cmake/Codegen.cmake:172 (message):</code>
</summary>

```
Dec 27 15:07:19 Traceback (most recent call last): 
Dec 27 15:07:19   File "/pytorch/cmake/../aten/src/ATen/gen.py", line 15, in <module> 
Dec 27 15:07:19     import preprocess_declarations 
Dec 27 15:07:19   File "/pytorch/aten/src/ATen/preprocess_declarations.py", line 3, in <module> 
Dec 27 15:07:19     from function_wrapper import TYPE_FORMAL_GENERIC 
Dec 27 15:07:19   File "/pytorch/aten/src/ATen/function_wrapper.py", line 1564 
Dec 27 15:07:19     <<<<<<< HEAD 
Dec 27 15:07:19      ^ 
Dec 27 15:07:19 SyntaxError: invalid syntax 
Dec 27 15:07:19 --  
Dec 27 15:07:19 CMake Error at cmake/Codegen.cmake:172 (message): 
Dec 27 15:07:19   Failed to get generated_cpp list 
Dec 27 15:07:19 Call Stack (most recent call first): 
Dec 27 15:07:19   caffe2/CMakeLists.txt:2 (include) 
Dec 27 15:07:19  
Dec 27 15:07:19  
Dec 27 15:07:19 -- Configuring incomplete, errors occurred! 
Dec 27 15:07:19 See also "/pytorch/build/CMakeFiles/CMakeOutput.log". 
Dec 27 15:07:19 See also "/pytorch/build/CMakeFiles/CMakeError.log". 
Dec 27 15:07:19 Traceback (most recent call last): 
Dec 27 15:07:19   File "setup.py", line 755, in <module> 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4074919) binary_linux_libtorch_2_7m_cpu_devtoolset7_shared-with-deps_build (2/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=78717310) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=78717310))
<details>
<summary>
<code>Dec 27 15:07:41 CMake Error at cmake/Codegen.cmake:172 (message):</code>
</summary>

```
Dec 27 15:07:41 Traceback (most recent call last): 
Dec 27 15:07:41   File "/pytorch/cmake/../aten/src/ATen/gen.py", line 15, in <module> 
Dec 27 15:07:41     import preprocess_declarations 
Dec 27 15:07:41   File "/pytorch/aten/src/ATen/preprocess_declarations.py", line 3, in <module> 
Dec 27 15:07:41     from function_wrapper import TYPE_FORMAL_GENERIC 
Dec 27 15:07:41   File "/pytorch/aten/src/ATen/function_wrapper.py", line 1564 
Dec 27 15:07:41     <<<<<<< HEAD 
Dec 27 15:07:41      ^ 
Dec 27 15:07:41 SyntaxError: invalid syntax 
Dec 27 15:07:41 --  
Dec 27 15:07:41 CMake Error at cmake/Codegen.cmake:172 (message): 
Dec 27 15:07:41   Failed to get generated_cpp list 
Dec 27 15:07:41 Call Stack (most recent call first): 
Dec 27 15:07:41   caffe2/CMakeLists.txt:2 (include) 
Dec 27 15:07:41  
Dec 27 15:07:41  
Dec 27 15:07:41 -- Configuring incomplete, errors occurred! 
Dec 27 15:07:41 See also "/pytorch/build/CMakeFiles/CMakeOutput.log". 
Dec 27 15:07:41 See also "/pytorch/build/CMakeFiles/CMakeError.log". 
Dec 27 15:07:41 Traceback (most recent call last): 
Dec 27 15:07:41   File "setup.py", line 755, in <module> 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4074940) binary_linux_libtorch_2_7m_cpu_gcc5_4_cxx11-abi_shared-with-deps_build (3/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=78717326) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=78717326))
<details>
<summary>
<code>Dec 27 15:07:53 CMake Error at cmake/Codegen.cmake:172 (message):</code>
</summary>

```
Dec 27 15:07:53 Traceback (most recent call last): 
Dec 27 15:07:53   File "/pytorch/cmake/../aten/src/ATen/gen.py", line 15, in <module> 
Dec 27 15:07:53     import preprocess_declarations 
Dec 27 15:07:53   File "/pytorch/aten/src/ATen/preprocess_declarations.py", line 3, in <module> 
Dec 27 15:07:53     from function_wrapper import TYPE_FORMAL_GENERIC 
Dec 27 15:07:53   File "/pytorch/aten/src/ATen/function_wrapper.py", line 1564 
Dec 27 15:07:53     <<<<<<< HEAD 
Dec 27 15:07:53      ^ 
Dec 27 15:07:53 SyntaxError: invalid syntax 
Dec 27 15:07:53 --  
Dec 27 15:07:53 CMake Error at cmake/Codegen.cmake:172 (message): 
Dec 27 15:07:53   Failed to get generated_cpp list 
Dec 27 15:07:53 Call Stack (most recent call first): 
Dec 27 15:07:53   caffe2/CMakeLists.txt:2 (include) 
Dec 27 15:07:53  
Dec 27 15:07:53  
Dec 27 15:07:53 -- Configuring incomplete, errors occurred! 
Dec 27 15:07:53 See also "/pytorch/build/CMakeFiles/CMakeOutput.log". 
Dec 27 15:07:53 See also "/pytorch/build/CMakeFiles/CMakeError.log". 
Dec 27 15:07:53 Traceback (most recent call last): 
Dec 27 15:07:53   File "setup.py", line 755, in <module> 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4074921) caffe2_onnx_py3_6_clang7_ubuntu16_04_build (4/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=78717327) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=78717327))
<details>
<summary>
<code>Dec 27 15:07:53 CMake Error at cmake/Codegen.cmake:172 (message):</code>
</summary>

```
Dec 27 15:07:53 -- Performing Test COMPILER_SUPPORTS_NO_AVX256_SPLIT - Failed 
Dec 27 15:07:53 Traceback (most recent call last): 
Dec 27 15:07:53   File "/var/lib/jenkins/workspace/cmake/../aten/src/ATen/gen.py", line 15, in <module> 
Dec 27 15:07:53     import preprocess_declarations 
Dec 27 15:07:53   File "/var/lib/jenkins/workspace/aten/src/ATen/preprocess_declarations.py", line 3, in <module> 
Dec 27 15:07:53     from function_wrapper import TYPE_FORMAL_GENERIC 
Dec 27 15:07:53   File "/var/lib/jenkins/workspace/aten/src/ATen/function_wrapper.py", line 1564 
Dec 27 15:07:53     <<<<<<< HEAD 
Dec 27 15:07:53      ^ 
Dec 27 15:07:53 SyntaxError: invalid syntax 
Dec 27 15:07:53 CMake Error at cmake/Codegen.cmake:172 (message): 
Dec 27 15:07:53   Failed to get generated_cpp list 
Dec 27 15:07:53 Call Stack (most recent call first): 
Dec 27 15:07:53   caffe2/CMakeLists.txt:2 (include) 
Dec 27 15:07:53  
Dec 27 15:07:53  
Dec 27 15:07:53 --  
Dec 27 15:07:53 -- Configuring incomplete, errors occurred! 
Dec 27 15:07:53 See also "/var/lib/jenkins/workspace/build/CMakeFiles/CMakeOutput.log". 
Dec 27 15:07:53 See also "/var/lib/jenkins/workspace/build/CMakeFiles/CMakeError.log". 
Dec 27 15:07:53 -- Building version 1.4.0a0+1a98619 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4074922) pytorch_linux_xenial_py2_7_9_build (5/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=78717378) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=78717378))
<details>
<summary>
<code>Dec 27 15:08:28 CMake Error at cmake/Codegen.cmake:172 (message):</code>
</summary>

```
Dec 27 15:08:28 Traceback (most recent call last): 
Dec 27 15:08:28   File "/var/lib/jenkins/workspace/cmake/../aten/src/ATen/gen.py", line 15, in <module> 
Dec 27 15:08:28     import preprocess_declarations 
Dec 27 15:08:28   File "/var/lib/jenkins/workspace/aten/src/ATen/preprocess_declarations.py", line 3, in <module> 
Dec 27 15:08:28     from function_wrapper import TYPE_FORMAL_GENERIC 
Dec 27 15:08:28   File "/var/lib/jenkins/workspace/aten/src/ATen/function_wrapper.py", line 1564 
Dec 27 15:08:28     <<<<<<< HEAD 
Dec 27 15:08:28      ^ 
Dec 27 15:08:28 SyntaxError: invalid syntax 
Dec 27 15:08:28 --  
Dec 27 15:08:28 CMake Error at cmake/Codegen.cmake:172 (message): 
Dec 27 15:08:28   Failed to get generated_cpp list 
Dec 27 15:08:28 Call Stack (most recent call first): 
Dec 27 15:08:28   caffe2/CMakeLists.txt:2 (include) 
Dec 27 15:08:28  
Dec 27 15:08:28  
Dec 27 15:08:28 -- Configuring incomplete, errors occurred! 
Dec 27 15:08:28 See also "/var/lib/jenkins/workspace/build/CMakeFiles/CMakeOutput.log". 
Dec 27 15:08:28 See also "/var/lib/jenkins/workspace/build/CMakeFiles/CMakeError.log". 
Dec 27 15:08:28 Traceback (most recent call last): 
Dec 27 15:08:28   File "setup.py", line 755, in <module> 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4074925) caffe2_onnx_py2_gcc5_ubuntu16_04_build (6/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=78717432) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=78717432))
<details>
<summary>
<code>Dec 27 15:08:58 CMake Error at cmake/Codegen.cmake:172 (message):</code>
</summary>

```
Dec 27 15:08:58 -- Performing Test COMPILER_SUPPORTS_NO_AVX256_SPLIT - Success 
Dec 27 15:08:58 Traceback (most recent call last): 
Dec 27 15:08:58   File "/var/lib/jenkins/workspace/cmake/../aten/src/ATen/gen.py", line 15, in <module> 
Dec 27 15:08:58     import preprocess_declarations 
Dec 27 15:08:58   File "/var/lib/jenkins/workspace/aten/src/ATen/preprocess_declarations.py", line 3, in <module> 
Dec 27 15:08:58     from function_wrapper import TYPE_FORMAL_GENERIC 
Dec 27 15:08:58   File "/var/lib/jenkins/workspace/aten/src/ATen/function_wrapper.py", line 1564 
Dec 27 15:08:58     <<<<<<< HEAD 
Dec 27 15:08:58      ^ 
Dec 27 15:08:58 SyntaxError: invalid syntax 
Dec 27 15:08:58 CMake Error at cmake/Codegen.cmake:172 (message): 
Dec 27 15:08:58   Failed to get generated_cpp list 
Dec 27 15:08:58 Call Stack (most recent call first): 
Dec 27 15:08:58   caffe2/CMakeLists.txt:2 (include) 
Dec 27 15:08:58  
Dec 27 15:08:58  
Dec 27 15:08:58 --  
Dec 27 15:08:58 -- Configuring incomplete, errors occurred! 
Dec 27 15:08:58 See also "/var/lib/jenkins/workspace/build/CMakeFiles/CMakeOutput.log". 
Dec 27 15:08:58 See also "/var/lib/jenkins/workspace/build/CMakeFiles/CMakeError.log". 
Dec 27 15:08:58 Traceback (most recent call last): 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4074937) pytorch_macos_10_13_py3_build (7/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=78717452) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=78717452))
<details>
<summary>
<code>Dec 27 07:09:01 CMake Error at cmake/Codegen.cmake:172 (message):</code>
</summary>

```
Dec 27 07:09:01 Traceback (most recent call last): 
Dec 27 07:09:01   File "/Users/distiller/project/cmake/../aten/src/ATen/gen.py", line 15, in <module> 
Dec 27 07:09:01     import preprocess_declarations 
Dec 27 07:09:01   File "/Users/distiller/project/aten/src/ATen/preprocess_declarations.py", line 3, in <module> 
Dec 27 07:09:01     from function_wrapper import TYPE_FORMAL_GENERIC 
Dec 27 07:09:01   File "/Users/distiller/project/aten/src/ATen/function_wrapper.py", line 1564 
Dec 27 07:09:01     <<<<<<< HEAD 
Dec 27 07:09:01      ^ 
Dec 27 07:09:01 SyntaxError: invalid syntax 
Dec 27 07:09:01 --  
Dec 27 07:09:01 CMake Error at cmake/Codegen.cmake:172 (message): 
Dec 27 07:09:01   Failed to get generated_cpp list 
Dec 27 07:09:01 Call Stack (most recent call first): 
Dec 27 07:09:01   caffe2/CMakeLists.txt:2 (include) 
Dec 27 07:09:01  
Dec 27 07:09:01  
Dec 27 07:09:01 -- Configuring incomplete, errors occurred! 
Dec 27 07:09:01 See also "/Users/distiller/project/build/CMakeFiles/CMakeOutput.log". 
Dec 27 07:09:01 See also "/Users/distiller/project/build/CMakeFiles/CMakeError.log". 
Dec 27 07:09:01 Traceback (most recent call last): 
Dec 27 07:09:01   File "setup.py", line 755, in <module> 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4074918) pytorch_linux_xenial_py3_6_gcc5_4_build (8/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=78717473) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=78717473))
<details>
<summary>
<code>Dec 27 15:09:10 CMake Error at cmake/Codegen.cmake:172 (message):</code>
</summary>

```
Dec 27 15:09:10 -- Performing Test COMPILER_SUPPORTS_NO_AVX256_SPLIT - Success 
Dec 27 15:09:10 Traceback (most recent call last): 
Dec 27 15:09:10   File "/var/lib/jenkins/workspace/cmake/../aten/src/ATen/gen.py", line 15, in <module> 
Dec 27 15:09:10     import preprocess_declarations 
Dec 27 15:09:10   File "/var/lib/jenkins/workspace/aten/src/ATen/preprocess_declarations.py", line 3, in <module> 
Dec 27 15:09:10     from function_wrapper import TYPE_FORMAL_GENERIC 
Dec 27 15:09:10   File "/var/lib/jenkins/workspace/aten/src/ATen/function_wrapper.py", line 1564 
Dec 27 15:09:10     <<<<<<< HEAD 
Dec 27 15:09:10      ^ 
Dec 27 15:09:10 SyntaxError: invalid syntax 
Dec 27 15:09:10 CMake Error at cmake/Codegen.cmake:172 (message): 
Dec 27 15:09:10   Failed to get generated_cpp list 
Dec 27 15:09:10 Call Stack (most recent call first): 
Dec 27 15:09:10   caffe2/CMakeLists.txt:2 (include) 
Dec 27 15:09:10  
Dec 27 15:09:10  
Dec 27 15:09:10 --  
Dec 27 15:09:10 -- Configuring incomplete, errors occurred! 
Dec 27 15:09:10 See also "/var/lib/jenkins/workspace/build/CMakeFiles/CMakeOutput.log". 
Dec 27 15:09:10 See also "/var/lib/jenkins/workspace/build/CMakeFiles/CMakeError.log". 
Dec 27 15:09:10 -- Building version 1.4.0a0+1a98619 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

gchanan committed:
Update on "Kill aten_custom_call."


Differential Revision: [D17172503](https://our.internmc.facebook.com/intern/diff/D17172503)
